docs/standard/translation.md

```diff
@@ -22,12 +22,15 @@ We currently actively maintain translations to:
 
 A number of other community translations exist in various levels of completion.
 
-## Translators and reviewers
+## Translators, proofreaders and reviewers
```

Comment:
We should potentially update this section to say something along the lines of:

"There are a number of roles in the translation process, including:

* Source language subject matter experts
* Translated language owner
* Translated language subject matter experts
* Translators
* Proofreaders
* Reviewers

"

and to provide the definition of the subject matter experts (noting that they do not necessarily translate full prose, just specific terminology.

Reply:
+ Terminologist

Reply:
The next commit will include a definition of the roles involved in the terminology process.